components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/pom.xml

```diff
@@ -181,6 +181,12 @@
             <artifactId>org.wso2.carbon.identity.application.mgt</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.user.api</groupId>
+            <artifactId>org.wso2.carbon.identity.rest.api.user.application.v1</artifactId>
+            <scope>provided</scope>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Why do we need this? If this is needed, make sure the application module comes first in the order.

Reply:
Addressed in [01e1307](https://github.com/wso2/identity-api-user/pull/129/commits/01e13074b77a93e08fd8c7d276b04f1525b274c5)

Reply:
@Shalindri why do we need this dependency? In the future we will be deprecating v1. Hence it is not ideal to use this dependency here.

Reply:
Addressed in [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)